examples/rendering-apis-with-htmx/README.md

```
@@ -1 +1 @@
-The quick start guide example with a rendering APIs with HTMX guide's changes.
+The quick start guide example with a rendering API and HTMX.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)